drivers/serial/uart_ns16550.c

```diff
@@ -32,7 +32,7 @@
 #include <uart.h>
 #include <sys_io.h>
 
-#include <drivers/serial/uart_ns16550.h>
+#include "uart_ns16550.h"
```

Comment:
This header shouldn't be private, as it exposes public values for use of the  NS16550 implementation of `uart_driver_api.drv_cmd`.  

Having said that, I am happy to remove `CMD_SET_DLF` as we already have a mechanism in place to hard-code the DLF value via DT ... and really, the value of the DLF is a function of the baud rate, and IMO shouldn't be manipulated independently at all, it should be accounted for in `set_baud_rate()`.

Reply:
well, it is include being included in the driver itself in the same directory. Are you saying application might need to access this value?

Reply:
Yes, `CMD_SET_DLF` is specifically for applications to use to control the driver, so the header should be left in `include/drivers/serial` rather than made local to the driver.  At least for now.

I'm happy to let this change go through as-is, and I'll file a github issue for myself to remove references to the `CMD_SET_DLF` and get rid of the header entirely.